tests/integration/cli_did_negative_test.go

```diff
@@ -0,0 +1,406 @@
+		Expect(err).To(BeNil())
+		Expect(res.Code).To(BeEquivalentTo(0))
+
+		// Second new valid DID Doc
+		did2 := "did:cheqd:" + network.DID_NAMESPACE + ":" + uuid.NewString()
+		keyId2 := did2 + "#key1"
+
+		pubKey2, privKey2, err := ed25519.GenerateKey(nil)
+		Expect(err).To(BeNil())
+
+		pubKeyMultibase582, err := multibase.Encode(multibase.Base58BTC, pubKey2)
+		Expect(err).To(BeNil())
+
+		payload2 := types.MsgCreateDidPayload{
+			Id: did2,
+			VerificationMethod: []*types.VerificationMethod{
+				{
+					Id:                 keyId2,
+					Type:               "Ed25519VerificationKey2020",
+					Controller:         did2,
+					PublicKeyMultibase: string(pubKeyMultibase582),
+				},
+			},
+			Authentication: []string{keyId2},
+		}
+
+		signInputs2 := []cli_types.SignInput{
+			{
+				VerificationMethodId: keyId2,
+				PrivKey:              privKey2,
+			},
+		}
+
+		// Fail to create a new DID Doc with missing cli arguments
+		//   a. missing payload, sign inputs and account
+		_, err = cli.CreateDid(types.MsgCreateDidPayload{}, []cli_types.SignInput{}, "")
+		Expect(err).ToNot(BeNil())
+
+		//   b. missing payload, sign inputs
+		_, err = cli.CreateDid(types.MsgCreateDidPayload{}, []cli_types.SignInput{}, testdata.BASE_ACCOUNT_2)
+		Expect(err).ToNot(BeNil())
+
+		//   c. missing payload, account
+		_, err = cli.CreateDid(types.MsgCreateDidPayload{}, signInputs2, "")
+
+		//   d. missing sign inputs, account
+		_, err = cli.CreateDid(payload2, []cli_types.SignInput{}, "")
+
+		//   e. missing payload
+		_, err = cli.CreateDid(types.MsgCreateDidPayload{}, signInputs2, testdata.BASE_ACCOUNT_2)
+		Expect(err).ToNot(BeNil())
+
+		//   f. missing sign inputs
+		_, err = cli.CreateDid(payload2, []cli_types.SignInput{}, testdata.BASE_ACCOUNT_2)
+		Expect(err).ToNot(BeNil())
+
+		//   g. missing account
+		_, err = cli.CreateDid(payload2, signInputs2, "")
+		Expect(err).ToNot(BeNil())
+
+		// Fail to create a new DID Doc with sign inputs mismatch
+		//   a. sign inputs mismatch
+		_, err = cli.CreateDid(payload2, signInputs, testdata.BASE_ACCOUNT_2)
+		Expect(err).ToNot(BeNil())
+
+		//   b. non-existing key id
+		_, err = cli.CreateDid(payload2, []cli_types.SignInput{
+			{
+				VerificationMethodId: "non-existing-key-id",
+				PrivKey:              privKey2,
+			},
+		}, testdata.BASE_ACCOUNT_2)
+		Expect(err).ToNot(BeNil())
+
+		//   c. non-matching private key
+		_, err = cli.CreateDid(payload2, []cli_types.SignInput{
+			{
+				VerificationMethodId: keyId2,
+				PrivKey:              privKey,
+			},
+		}, testdata.BASE_ACCOUNT_2)
+
+		// Fail to create a new DID Doc with non-supported VM type
+		payload3 := payload2
+		payload3.VerificationMethod[0].Type = "NonSupportedVMType"
+		_, err = cli.CreateDid(payload3, signInputs2, testdata.BASE_ACCOUNT_2)
+		Expect(err).ToNot(BeNil())
+
+		// Fail to create a new DID Doc with already existing DID
+		_, err = cli.CreateDid(payload, signInputs, testdata.BASE_ACCOUNT_1)
+		Expect(err).ToNot(BeNil())
+	})
```

Comment:
This whole section seems like different scenarios/use cases, and also different than the one on top. Describing it as a comment is not helping distinguish differences in these test cases, as far as I can see. What am I missing here?